R/zzz.R

```diff
@@ -16,3 +16,21 @@ bcmaps_env <- new.env(parent = emptyenv())
   assign("bcmaps_update_message", FALSE, envir = bcmaps_env)
 }
 
+
+bcmapsStartupMessage <- function() {
+  msg <- "Support for Spatial objects (`sp`) was deprecated in {bcmaps} v1.2.0, and will be removed altogether in the Summer 2023. Please use `sf` objects with {bcmaps}."
+}
+
+.onAttach <- function(lib, pkg)
+{
+  # startup message
+  msg <- bcmapsStartupMessage()
+  packageStartupMessage(msg)
+  invisible()
+}
+
+deprecate_sp <- function(what){
+  lifecycle::deprecate_warn( when = "1.2.0",
+                             what = what,
+                             details = "Please use `sf` objects with {bcmaps}, support for Spatial objects (sp) will be removed in Summer 2023.")
```

Comment:
Same as above.

Reply:
Not sure what same this comment is referring to? This is the deprecate wrapper function we are using throughout to deprecate sp?